infra/docker/testground-redis-exporter/Dockerfile

```diff
@@ -1,5 +0,0 @@
-FROM bitnami/redis-exporter
```

Comment:
@coryschwartz I think this file is not used, so I am removing it.

Reply:
Yes, I think this was added unintentionally.